TricksterBots/Bots/Bridge/bridgebid/conventions/Stayman.cs

```diff
@@ -183,6 +183,7 @@ private static bool InterpretResponderRebid(InterpretedBid rebid)
             if (rebid.declareBid.level == 3 && BridgeBot.IsMinor(rebid.declareBid.suit))
             {
                 //  (SAYC Booklet): if responder rebids three of either minor, he shows slam interest and at least a five-card suit
+                // TODO: Is this IFF declarer has "hit" the responder's suit?    Don't understand the logic here...
```

Comment:
So far, I haven't found where this logic originated from. If it doesn't make sense, it may have been a mistake (possibly copied from somewhere else when initially setting up the code). I'm okay replacing/removing it (and seeing what impact, if any, it has on the test suite).